### PR TITLE
Added information about frokostseminar to DevOpsDays Oslo frontpage

### PR DESCRIPTION
**Pull request by antonbabenko**

### CODE REVIEW

File: content/events/2016-oslo/welcome.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ aliases = ["/events/2016-oslo"]
 
 <p>At a high-level, each day will consist of a mix of presentations, ignite presentations, and open spaces. There will be an evening party for all participants scheduled for the evening of September 5th. A great opportunity for networking, mingling and having a good time.</p>
 
+<h3>Frokostseminarer om DevOps og smidig drift</h3>
+<p>Redpill Linpro is organizing DevOps breakfast seminar 7th of June. The program consists of presentations both for Devs and Ops. There will be these talks:
```
Comment (phrawzty) [inline content/events/2016-oslo/welcome.md]: Please close this paragraph tag.

File: content/events/2016-oslo/welcome.md [hunk 2]
```diff
@@ -18,6 +18,18 @@ aliases = ["/events/2016-oslo"]
...
+
+<li> Don't wait for JDK9, develop modular Java software today with OSGi</li>
+<li> Docker orchestration with Ansible in practice</li>
+<li> A presentations about Atlasian Bamboo</li>
+
+<p>You can find more details and RSVP on <a href="http://www.meetup.com/Redpill-Linpro-frokostseminar-smidig-drift/events/230340363/" target="_blank">event page</a>
```
Comment (phrawzty) [inline content/events/2016-oslo/welcome.md]: Please close this paragraph tag.

File: content/events/2016-oslo/welcome.md [hunk 3]
```diff
@@ -18,6 +18,18 @@ aliases = ["/events/2016-oslo"]
...
+
+<p>This breakfast seminar is for IT Managers, CIOs, system administrators, system architects, developers, DevOps, DBAs, infrastructure consultants and others interested in DevOps, deveopment and agile operations.
```
Comment (phrawzty) [inline content/events/2016-oslo/welcome.md]: Please close this paragraph tag.

File: content/events/2016-oslo/welcome.md [hunk 4]
```diff
@@ -18,6 +18,18 @@ aliases = ["/events/2016-oslo"]
 
 <p>At a high-level, each day will consist of a mix of presentations, ignite presentations, and open spaces. There will be an evening party for all participants scheduled for the evening of September 5th. A great opportunity for networking, mingling and having a good time.</p>
 
+<h3>Frokostseminarer om DevOps og smidig drift</h3>
+<p>Redpill Linpro is organizing DevOps breakfast seminar 7th of June. The program consists of presentations both for Devs and Ops. There will be these talks:
+
+<li> Don't wait for JDK9, develop modular Java software today with OSGi</li>
```
Comment (phrawzty) [inline content/events/2016-oslo/welcome.md]: Accidental space between the list tag and the content? Idem for next two lines.

Comment (phrawzty) [inline content/events/2016-oslo/welcome.md]: List elements [must be contained](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li) within list blocks.

**Comment by phrawzty** [comment]
You could fix the HTML, or you could replace it all with Markdown (this is a `.md` document after all). Your call. 😄

**Comment by antonbabenko** [comment]
Fixed

**Comment by antonbabenko** [comment]
Thanks Dan!

**Pull request merged**